haproxy/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - haproxy
 
+1.1.0 / 2017-06-08
```

Comment:
That is correct, we can bump those later. We should bump the version, as you're already doing.